# Config for teleport script
places:
  home: {
    'robot_pose': [7.087382,-8.720616,-116.360490,0.011443,0.060502,0.304482],
    'camera_pose': [7.087382,-8.720616,-116.360490,0.011443,0.060502,0.304482]
  }
  artillery: {
    'robot_pose': [-150.113998,97.727378,-155.246244,-0.011886,0.016902,1.687642],
    'camera_pose': [-150.113998,97.727378,-155.246244,-0.011886,0.016902,1.687642]
  }
  mud: {
    'robot_pose': [-66.099145,-208.337754,-145.292877,-0.010855,0.030645,-3.086439],
    'camera_pose': [-66.099145,-208.337754,-145.292877,-0.010855,0.030645,-3.086439]
  }
